### PR TITLE
Ef core

### PR DESCRIPTION
**Pull request by SarahVestergren**

### CODE REVIEW

**Review by baggekjr — APPROVED** [review]
Perf :)

**Comment by AnnikaJensby** [comment]
very nice

**Review by baggekjr — APPROVED** [review]
Just okay

File: src/Chirp.Razor/Chirp.Razor.csproj [hunk 1]
```diff
@@ -14,10 +14,5 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="7.0.11" />
   </ItemGroup>
 
-  <ItemGroup>
```
Comment (ClaraSoenderborg) [inline src/Chirp.Razor/Chirp.Razor.csproj:17]: Nice cleanup :)

**Review by JuliaKlompmaker — COMMENTED** [review]
nice

**Review by ClaraSoenderborg — APPROVED** [review]
Great work. Team work! :)

**Review by JuliaKlompmaker — APPROVED** [review]

**Review by ClaraSoenderborg — APPROVED** [review]

**Pull request merged**